app-benchmarks/unigine-superposition/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
```

Comment:
This isn't a proxied package, remove `proxied="yes"`

Reply:
Thanks, done 578adc595